_component_design/toast.md

```diff
@@ -5,40 +5,136 @@ layout: component
 category: Components
 usage: >
   Toasts are a specialized notification component used to provide feedback on the status of an actions taking place in the background. Toasts are meant to be ephemeral and capture users attention by having the highest Z-index of all components.
-preview-image: components/preview-images/toast.svg
-status: deprecated
-resource: false
-last-modified: 2017-08-17
+preview-image: preview-images/toast.svg
+status: stable
+resource: true
+last-modified: 2017-11-05
 ---
 
 {% include toc.html %}
 
+## Introductions
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Default"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- A toast notification provides information for action outside of the user’s current context. User understand that clicking on any links within the toast notification will take them out of the current context. 
```

Comment:
do you mean "information that a user can act upon"?
Users understand that...
(do they already understand this, or do you mean they should understand this?)

Reply:
Reworded to, "- A toast notification provides information that a user can act upon. Toast incorporate an optional tertiary CTA that navigate the user out of the current page or context. "